src/stores/game.js

```diff
@@ -334,6 +339,20 @@ export const useGameStore = defineStore('game', {
         });
       });
     },
+    async setIsRanked({ isRanked }) {
```

Comment:
Sorry to change this again, but our other actions that make requests start with `request`. Let's change this to be consistent
```suggestion
    async requestSetIsRanked({ isRanked }) {
```